docs/examples/viz_parametric_surfaces_v2.py

```diff
@@ -0,0 +1,250 @@
+###############################################################################
+# Importing necessary modules
+
+from fury import window, actor, ui
+import numpy as np
+from scipy.spatial import Delaunay
+
+###############################################################################
+# Function that returns the parametric surface specified by the user.
+
+
+def parametric_func(name, colors=(1, 1, 1), coords=[0, 0, 0], scale=1.0):
+    """
+    Parameters
+    ----------
+    name : string - "mobius_strip" or "kleins_bottle" or "roman_surface"
+        or "boys_surface" or "bohemian_dome" or "dinis_surface" or
+        "pluckers_conoid" or "bours_surface". Determines the parametric surface
+        to be returned.
+    colors : ndarray (3) or tuple (3)
+        Determines the color of the surface. (default : white)
+    coords : ndarray (3)
+        Determines the coordinates where the parametric surface will be
+        rendered. (default : [0, 0, 0])
+    opacity : float, optional
+        Takes values from 0 (fully transparent) to 1 (opaque). (default : 1)
+    scale : float, optional
+        Determines the size of the parametric surface returned. (default : 1)
+    cmap : bool, optional
+        Set it to true if colormap is being mapped to the surface.
+    cmap_name : string, optional
+        Stores the name of the colormap specified by the user, cmap should be
+        set to True for the colormap to be rendered.
+
+    Returns
+    -------
+    vtkActor
+
+    """
+    def sin(x):
+        return np.sin(x)
+
+    def cos(x):
+        return np.cos(x)
+
+    npoints = 400
+
+    def scaler(arr):
+        arr = np.array(arr)
+        maxima = np.max(arr)
+        minima = np.min(arr)
+        return (arr - maxima)/(maxima - minima)
+
+    if (name == "mobius_strip"):
+        u = np.linspace(0, 2*np.pi, npoints)
+        v = np.linspace(-1, 1, npoints)
+        u, v = np.meshgrid(u, v)
+        u = u.reshape(-1)
+        v = v.reshape(-1)
+        points2D = np.vstack([u, v]).T
+        tri = Delaunay(points2D)
+
+        x = coords[0] + scale*(1 + v/2 * cos(u/2)) * cos(u)
+        y = coords[1] + scale*(1 + v/2 * cos(u/2)) * sin(u)
+        z = coords[2] + scale*v/2 * sin(u/2)
+
+        xyz = np.array([(a, b, c) for (a, b, c) in zip(x, y, z)])
```

Comment:
each element is a primitive for us. you should look at `fury.primitive`.  Every primitive returns vertices and faces. Then, the actor call the primitive and build the actor.

You should look at `actor.square` or `actor.superquadrics` to understand how it works and reproduce it with your case. Let us know if you have any question